File: app/src/main/java/com/grace/placessearch/data/model/Item.java
package com.grace.placessearch.data.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;
import java.util.List;

public class Item {

    @SerializedName("id")
    @Expose
    private String id;
    @SerializedName("name")
    @Expose
    private String name;
    @SerializedName("categories")
    @Expose
    private List<Category> categories = new ArrayList<>();
    @SerializedName("allDay")
    @Expose
    private boolean allDay;
    @SerializedName("date")
    @Expose
    private long date;
    @SerializedName("timeZone")
    @Expose
    private String timeZone;
    @SerializedName("summary")
    @Expose
    private String summary;
    @SerializedName("text")
    @Expose
    private String text;
    @SerializedName("images")
    @Expose
    private List<String> images = new ArrayList<>();
    @SerializedName("stats")
    @Expose
    private Stats stats;
    @SerializedName("participants")
    @Expose
    private List<Participant> participants = new ArrayList<>();
    @SerializedName("genres")
    @Expose
    private String genres;
    @SerializedName("rating")
    @Expose
    private String rating;
    @SerializedName("runningTime")
    @Expose
    private String runningTime;

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public List<Category> getCategories() {
        return categories;
    }

    public void setCategories(List<Category> categories) {
        this.categories = categories;
    }

    public boolean isAllDay() {
        return allDay;
    }

    public void setAllDay(boolean allDay) {
        this.allDay = allDay;
    }

    public long getDate() {
        return date;
    }

    public void setDate(long date) {
        this.date = date;
    }

    public String getTimeZone() {
        return timeZone;
    }

    public void setTimeZone(String timeZone) {
        this.timeZone = timeZone;
    }

    public String getSummary() {
        return summary;
    }

    public void setSummary(String summary) {
        this.summary = summary;
    }

    public String getText() {
        return text;
    }

    public void setText(String text) {
        this.text = text;
    }

    public List<String> getImages() {
        return images;
    }

    public void setImages(List<String> images) {
        this.images = images;
    }

    public Stats getStats() {
        return stats;
    }

    public void setStats(Stats stats) {
        this.stats = stats;
    }

    public List<Participant> getParticipants() {
        return participants;
    }

    public void setParticipants(List<Participant> participants) {
        this.participants = participants;
    }

    public String getGenres() {
        return genres;
    }

    public void setGenres(String genres) {
        this.genres = genres;
    }

    public String getRating() {
        return rating;
    }

    public void setRating(String rating) {
        this.rating = rating;
    }

    public String getRunningTime() {
        return runningTime;
    }

    public void setRunningTime(String runningTime) {
        this.runningTime = runningTime;
    }

}

File: app/src/main/kotlin/com/grace/placessearch/common/data/model/Item.kt
package com.grace.placessearch.common.data.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import java.util.*

class Item {

    @SerializedName("id")
    @Expose
    var id: String? = null
    @SerializedName("name")
    @Expose
    var name: String? = null
    @SerializedName("categories")
    @Expose
    var categories: List<Category> = ArrayList()
    @SerializedName("allDay")
    @Expose
    var isAllDay: Boolean = false
    @SerializedName("date")
    @Expose
    var date: Long = 0
    @SerializedName("timeZone")
    @Expose
    var timeZone: String? = null
    @SerializedName("summary")
    @Expose
    var summary: String? = null
    @SerializedName("text")
    @Expose
    var text: String? = null
    @SerializedName("images")
    @Expose
    var images: List<String> = ArrayList()
    @SerializedName("stats")
    @Expose
    var stats: Stats? = null
    @SerializedName("participants")
    @Expose
    var participants: List<Participant> = ArrayList()
    @SerializedName("genres")
    @Expose
    var genres: String? = null
    @SerializedName("rating")
    @Expose
    var rating: String? = null
    @SerializedName("runningTime")
    @Expose
    var runningTime: String? = null

}